src/main/java/org/stellar/sdk/Operation.java

```diff
@@ -209,6 +209,12 @@ public static Operation fromXdr(
       case INVOKE_HOST_FUNCTION:
         operation = InvokeHostFunctionOperation.fromXdr(body.getInvokeHostFunctionOp());
         break;
+      case BUMP_FOOTPRINT_EXPIRATION:
+        operation = BumpFootprintExpirationOperation.fromXdr(body.getBumpFootprintExpirationOp());
```

Comment:
I think we should throw a NotSupportException here for invoke/bump/restore for now and add a TODO referring to https://github.com/stellar/java-stellar-sdk/issues/495

Reply:
We will not merge into the master branch now, but into the soroban branch instead. We can assume that all necessary work will be completed before merging the code into the main branch. Therefore, I prefer to track these TODO items in the issue rather than in the code itself. What do you think?

Reply:
yes, the feature branch and no-one else is developing on it or using it incrementally allows for that and will leave to your discretion. On it's own, It looks like it's complete but reality is not, so, adding the NotSupportedFunction here in code and a unit test that asserts that as outcome for now is just a way to just make this clear right in the code that it is currently not complete.

Reply:
OK, actually I have been working on this related task in the soroban-server branch, so let's keep it that way for now.

Reply:
ok, understood, that is https://github.com/stellar/java-stellar-sdk/pull/492 correct, it will be first to reference this aspect?